### ISSUE TITLE
about the loss function

### ISSUE THREAD

**Issue by sibangde** [ninetf135246/pytorch-Learning-to-See-in-the-Dark#4]
I want to use pytorch to implement a new loss function. And almost all the online information say that I should make a new class that inherit the nn.Module. But I found you just create a function ‘reduce_mean()’. Does it means that it is just so easy in pytorch to make an own funtion.

**Comment by ninetf135246** [comment]
Hi, @sibangde ,
Yes.
BTW, the question you're asking is not an issue at all. Please go to PyTorch forum if you have any question about Pytorch.
